src/controllers/project.js

```diff
@@ -0,0 +1,108 @@
+/* eslint no-underscore-dangle: ["error", { "allow": ["_id"] }] */
+const Project = require('../models/project');
+
+const getAllProjects = async (req, res, next) => {
```

Comment:
okay no we need to test all these controller functions and then its ready to be merged